tenable/io/v3/assets.py

```diff
@@ -0,0 +1,289 @@
+"""
+Assets
+======
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`assets <assets>` API endpoints.
+
+Methods available on ``tio.v3.assets``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AssetsAPI
+    :members:
+"""
+from tenable.io.v3.base.endpoints.uw import UWBaseEndpoint
+
+
+class AssetsAPI(UWBaseEndpoint):
+    """
+    This will contain all methods related to Assets
+    """
```

Comment:
An attribute `_path` should be defined on the class with the string value of "v3/assets".  This refers to the base path that this resource is on.

Reply:
An attribute `_conv_json` should be defined with a bool value of `True`.  This informs the HTTP method methods to attempt automatic conversion of json responses.